examples/drupal_loadtest.rs

```diff
@@ -81,19 +81,27 @@ fn main() {
 }
 
 /// View the front page.
-async fn drupal_loadtest_front_page(client: &mut GooseClient) {
-    let mut response = client.get("/").await;
+async fn drupal_loadtest_front_page(client: &GooseClient) {
+    let mut response = client.get("/", None).await;
 
     // Grab some static assets from the front page.
     match response.response {
         Ok(r) => match r.text().await {
             Ok(t) => {
                 let re = Regex::new(r#"src="(.*?)""#).unwrap();
+                // Collect copy of URLs to run them async
+                let mut urls = Vec::new();
+                let mut futures = Vec::new();
                 for url in re.captures_iter(&t) {
                     if url[1].contains("/misc") || url[1].contains("/themes") {
-                        let _response = client.set_request_name("static asset").get(&url[1]);
```

Comment:
Was this a bug with missing await?

Reply:
It seems so? I've not yet dug into why we were still making static requests -- at first glance it seems we shouldn't have been making any. 

I think the bug was masked because of how `.set_request_name()` worked -- the request name would have just been consumed by the _next_ request. This would invalidate some of our earlier numbers as well, as it would suggest that prior to this fix we were not actually loading static elements in this example.

Reply:
I've patched `drupal_loadtest` and am re-running the pre-patch tests:

```rust
diff --git a/examples/drupal_loadtest.rs b/examples/drupal_loadtest.rs
index a34a0c4..fb4e45f 100644
--- a/examples/drupal_loadtest.rs
+++ b/examples/drupal_loadtest.rs
@@ -89,11 +89,16 @@ async fn drupal_loadtest_front_page(client: &mut GooseClient) {
         Ok(r) => match r.text().await {
             Ok(t) => {
                 let re = Regex::new(r#"src="(.*?)""#).unwrap();
+                // Collect copy of URLs to run them async
+                let mut urls = Vec::new();
                 for url in re.captures_iter(&t) {
                     if url[1].contains("/misc") || url[1].contains("/themes") {
-                        let _response = client.set_request_name("static asset").get(&url[1]);
+                        urls.push(url[1].to_string());
                     }
                 }
+                for index in 0..urls.len() {
+                    client.set_request_name("static asset").get(&urls[index]).await;
+                }
             }
             Err(e) => {
                 eprintln!("failed to parse front page: {}", e);
```

Reply:
Confirmed that this was a bug in the drupal_loadtest example that has been there since we converted to using async.